src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -33,7 +33,7 @@ public void Initialize(EntityInfo entity, Func<IncomingMessageDetails, ReceiveCo
             receiveMode = settings.Get<ReceiveMode>(WellKnownConfigurationKeys.Connectivity.MessageReceivers.ReceiveMode);
 
             incomingCallback = callback;
-            this.errorCallback = errorCallback;
+            this.errorCallback = errorCallback ?? EmptyErrorCallback;
```

Comment:
Discussed this ( @danielmarbach & @boblangley ) and we decided to leave it as-is to keep it close to what it was. This will not be an issue when internalization takes place.